apiml-security-common/src/test/java/org/zowe/apiml/security/common/login/LoginFilterTest.java

```diff
@@ -86,11 +92,18 @@ void shouldCallAuthenticationManagerAuthenticateWithAuthHeader() throws ServletE
         httpServletRequest.addHeader(HttpHeaders.AUTHORIZATION, VALID_AUTH_HEADER);
         httpServletResponse = new MockHttpServletResponse();
 
+        AtomicBoolean called = new AtomicBoolean(false);
```

Comment:
This is used three times. It could be a method

Reply:
I'll make this refactor